pkg/apiutil/apiutil.go

```diff
@@ -68,3 +65,22 @@ func ReadJSON(r io.ReadCloser, data interface{}) error {
 
 	return err
 }
+
+// FieldError connects an error to a particular field
+type FieldError struct {
+	error
+	field string
+}
+
+// VarsParseUint wraps strconv.ParseUint with detailed error reporting
+func VarsParseUint(vars map[string]string, varName string, base int, bitsize int) (uint64, error) {
```

Comment:
may ParseUintField be better? here you return FieldError too.

Btw, I find that you all use base 10 and bitsize 64 for this function, do we need to care other base and bitsize? I think mostly using 10 and 64 is enough.